src/integration-test/java/com/commercetools/sync/integration/externalsource/products/ProductSyncIT.java

```diff
@@ -455,9 +463,11 @@ public void sync_withSingleBatchSyncing_ShouldSync() {
         final ProductSync productSync = new ProductSync(syncOptions);
         final ProductSyncStatistics syncStatistics = executeBlocking(productSync.sync(batch));
 
-        assertThat(syncStatistics.getReportMessage())
-            .isEqualTo(format("Summary: %d products were processed in total (%d created, %d updated and %d products"
-                + " failed to sync).", 5, 4, 1, 0));
+        assertThat(syncStatistics.getProcessed()).isEqualTo(5);
+        assertThat(syncStatistics.getCreated()).isEqualTo(4);
+        assertThat(syncStatistics.getUpdated()).isEqualTo(1);
+        assertThat(syncStatistics.getFailed()).isEqualTo(0);
```

Comment:
having so long similar `assertThat()` chains i'd make some util `assertSyncStatistic()` method, which accepts a statistic instance and a list of expected values.

Reply:
9a6d987, 474c8c4